src/ng/directive/ngModel.js

```diff
@@ -921,7 +924,8 @@ function setupModelWatcher(ctrl) {
     }
 
     return modelValue;
```

Comment:
Can this be deleted now?

Reply:
I guess we still need to return this because we need the digest to cycle again if the model value has actually changed...?

Reply:
Yeah this comment is outdated now. This was from when this was moved to the listener fn...